script.js

```diff
@@ -1,12 +1,22 @@
 const stuffQuizProxiedURL = 'https://corsproxy.io/?' + encodeURIComponent('https://www.stuff.co.nz/_json/national/quizzes?limit=99');
+import { initializeApp } from "https://www.gstatic.com/firebasejs/10.8.0/firebase-app.js";
+import { addDoc, collection, getDocs, getFirestore, onSnapshot, query, updateDoc, where } from "https://www.gstatic.com/firebasejs/10.8.0/firebase-firestore.js";
```

Comment:
I feel strongly about this 😅 

Move the imports above the url const please 🙏

Reply:
Whoops! Good spotting. Updated with commit [160b8f5](https://github.com/AlDrag/quizzaroo/pull/14/commits/160b8f5d2d40329c7dd4378d179e770b5e22cae6).